lib/cosmos/dart/processes/dart_decom_server.rb

```diff
@@ -0,0 +1,39 @@
+# encoding: ascii-8bit
+
+# Copyright 2018 Ball Aerospace & Technologies Corp.
+# All Rights Reserved.
+#
+# This program is free software; you can modify and/or redistribute it
+# under the terms of the GNU General Public License
+# as published by the Free Software Foundation; version 3 with
+# attribution addendums as found in the LICENSE.txt
+
+require File.expand_path('../../config/environment', __FILE__)
+require 'dart_common'
+require 'dart_decom_query'
+
+Cosmos.catch_fatal_exception do
+  DartCommon.handle_argv
+
+  Cosmos::Logger.level = Cosmos::Logger::INFO
+  dart_logging = DartLogging.new('dart_decom_server')
```

Comment:
I'd like to move the logging into DartDecomQuery like with DartTcpipServerInterface but didn't know when / where to call the logging.stop method. Or maybe we move the logging out of DartTcpipServerInterface into dart_stream_server to match dart_decom_server. I don't like the inconsistency right now.